source/_sass/infeedl.scss

```diff
@@ -63,9 +63,16 @@ h1, h2, h3, h4, h5, h6 {
   li {
     a {
       text-transform: none;
-      color: #fff;
+      color: #F5A623;
       font-size: .9em;
       font-weight: normal;
+      border-bottom:1px solid rgba(251,192,57,.5);
```

Comment:
Commas in function arguments should be followed by a single space<br>Colon after property should be followed by one space